internal/ir/xds.go

```diff
@@ -2094,15 +2094,6 @@ type GlobalRateLimit struct {
 
 	// Rules for rate limiting.
 	Rules []*RateLimitRule `json:"rules,omitempty" yaml:"rules,omitempty"`
```

Comment:
you'll need to add a new field called `Name` under RateLimitRule

and the naming could look like `default/policy-for-route/rule/0` which is `<policy-ns>/<policy-namespace>/rule/<rule-index>